server/utils/assessments/isAssessment.ts

```diff
@@ -0,0 +1,8 @@
+import {
+  ApprovedPremisesApplication as Application,
+  ApprovedPremisesAssessment as Assessment,
+} from '../../@types/shared'
+
+export default (applicationOrAssessment: Application | Assessment): applicationOrAssessment is Assessment => {
+  return 'allocatedToStaffMemberId' in applicationOrAssessment
```

Comment:
This has just changed (ref https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/546) as an assessment no longer has an `allocatedToStaffMemberId`